codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/ConvenienceTypeOverload.java

```diff
@@ -21,7 +21,7 @@
 /**
  * Customization to allow generation of additional setter overloads for a 'convenience' type (i.e. a
```

Comment:
Can this go away now? I believe SdkBytes solves the problem this customization was trying to address.